Realm/Realm/Native/NativeCommon.cs

```diff
@@ -51,17 +46,16 @@ private static unsafe void DebugLogger(byte* utf8String, IntPtr stringLen)
         public static extern void set_debug_logger(DebugLoggerCallback callback);
 #endif  // DEBUG
 
-        [DllImport(InteropConfig.DLL_NAME, EntryPoint = "register_callbacks", CallingConvention = CallingConvention.Cdecl)]
-        public static extern void register_callbacks(NotifyRealmCallback notifyRealmCallback, GetNativeSchemaCallback nativeSchemaCallback);
-
         [DllImport(InteropConfig.DLL_NAME, EntryPoint = "delete_pointer", CallingConvention = CallingConvention.Cdecl)]
         public static extern unsafe void delete_pointer(void* pointer);
 
         [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_reset_for_testing", CallingConvention = CallingConvention.Cdecl)]
         public static extern void reset_for_testing();
 
-        public static unsafe void Initialize()
+        private static unsafe void InitializeOnce()
```

Comment:
Can't we move this to `Initialize`? Don't see much value in keeping them separate.